test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+
+        public static string GetNpmPrefixPath()
+        {
+            string npmPath = GetNpmExecutablePath();
+            string npmPrefixPath = RunCommand(npmPath, "-g root");
+
+            return npmPrefixPath.Trim();
+        }
+
+        private static string RunCommand(string command, string arguments, int timeoutMilliseconds = 30000)
+        {
+            try
+            {
+                using (Process process = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = command,
+                        Arguments = arguments,
+                        RedirectStandardOutput = true,
+                        RedirectStandardError = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true,
+                    }
+                })
+                {
+                    process.Start();
+                    string output = process.StandardOutput.ReadToEnd();
+                    string errorOutput = process.StandardError.ReadToEnd();
+                    process.WaitForExit(timeoutMilliseconds);
+
+                    int exitCode = process.ExitCode;
+                    if (exitCode != 0)
+                    {
+                        throw new ApplicationException($"Command exited with code {exitCode}. Error: {errorOutput}");
+                    }
+
+                    return output;
+                }
+            }
+            catch (Exception)
+            {
+                throw new NpmNotFoundException();
+            }
+        }
+
+        private static string GetNpmExecutablePath()
+        {
+            string commandName = IsWindows() ? "where" : "which";
+            string result = RunCommand(commandName, "npm");
+
+            string npmPath;
+
+            string[] lines = result?.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries);
+
+            if (IsWindows())
+            {
+                npmPath = lines?.FirstOrDefault(line => !string.IsNullOrWhiteSpace(line) && line.EndsWith("npm.cmd"));
+            }
+            else
+            {
+                npmPath = lines?.FirstOrDefault(line => !string.IsNullOrWhiteSpace(line));
+            }
+
+            if (string.IsNullOrWhiteSpace(npmPath))
+            {
+                throw new NpmNotFoundException("NPM executable not found. Please make sure the NPM executable is installed and check the configured path.");
```

Comment:
and check the configured PATH environment variable

Reply:
also the default message would work here

Reply:
went for the first comment